app/controllers/hours_controller.rb

```diff
@@ -1,5 +1,5 @@
 class HoursController < ApplicationController
-  before_action :set_hour, only: [:show, :edit, :update, :destroy]
+  before_action :set_hour, only: [:show, :edit, :update, :destroy, :create_redirect]
```

Comment:
the before action should  rather run at `:create` instead of `:create_redirect`, because thats the context where you want hour to be set as class variable

Reply:
Not so sure, because before `:create` hour is not saved (so find crashes). When `:create_redirect` is called, hour is already saved.

Reply:
true